lib/manager/actions/status.js

```diff
@@ -210,11 +211,15 @@ export function startJobMonitor (showMonitor: boolean = true) {
     const timerFunction = () => dispatch(checkJobStatus())
```

Comment:
This function doesn't look like it's needed anymore.  Couldn't the dispatch be used directly?

Reply:
Yea, I suppose we might reduce obfuscation by eliminating the `timerFunction`.